fontData.js

```diff
@@ -192,6 +199,21 @@ const buildChars = font => {
     }
   }
 
+  // List all chars not grouped under scripts in a misc category
+  // Also, ignore 0xFFFF which is erroneously reported as a char
+  // by Fontkit
+  const uncategorisedChars = fontCharset.filter(
```

Comment:
Can you add a test for this please?

Reply:
There's probably an opportunity here to do this all in one pass; loop over all chars in the font and assign the correct category or mark uncategorised.
For now I guess this is fine as this processing is not done very often. In WF it is probably worth it to refactor this to reduce to a single pass.

Reply:
Added tests. Needed a new test font for this.

I haven't ported this to WF yet. (Wouldn't have to if we put this in the engine! ✊ ) But agree that it shouldn't be this crude then.